pkg/gadgets/trace/exec/tracer/tracer_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	"path"
 	"sort"
 	"strings"
+	"syscall"
```

Comment:
```suggestion
	"golang.org/x/sys/unix"
```
The documentation for [syscall](https://pkg.go.dev/syscall) says:
> most new code should prefer that package ["golang.org/x/sys/unix"] where possible

And that is what was done in other files:
```bash
$ git grep Gettid
```

Reply:
DONE